pkg/source/control/kafkasource/message.go

```diff
@@ -0,0 +1,89 @@
+package control
```

Comment:
should this be more in "common", rather than in source ? For reuse?

Reply:
No, this part is the part that depends on the component we're going to control. It contains the opcodes and messages specific to the "component" controlled uses during the communication, like the contract message payload, the status message payload etc.